test/runtests.jl

```diff
@@ -150,6 +150,11 @@ end
         @test size(embs_fr.embeddings) == (300, 100)
     end
 
+    @testset_nokeep_data "Spanish" begin
```

Comment:
If we add testsets for each of these then out TravisCI tests will never complete in time.

Reply:
That's true, but is there another way to check if the checksum is correctly provided at the correct location? Maybe in such PRs, we could ask them to remove the test set added, once the CI tests pass successfully, preventing new tests from getting merged.